contrib/zmq/zmq_sub.py

```diff
@@ -39,6 +40,9 @@
         elif topic == "rawtx":
             print '- RAW TX ('+sequence+') -'
             print binascii.hexlify(body)
+        elif topic == "checkedblock":
+            print ' - CHECKED BLOCK ('+sequence+') -'
```

Comment:
Delete the initial space, for consistency with the other topics.

Reply:
Good catch!